deep-learning/_posts/2018-07-23-cascade-rcnn.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: review
+title: "Cascade R-CNN: Delving into High Quality Object Detection"
+tags: deep-learning CNN localization
+author: Frédéric Branchaud-Charron
+cite:
+    authors: "Zhaowei Cai, Nuno Vasconcelos"
+    title:   "Cascade R-CNN: Delving into High Quality Object Detection"
+    venue:   "CVPR 2018"
+pdf:   "https://arxiv.org/pdf/1712.00726.pdf"
+---
+
+The basic idea of this paper is to train multiple detection heads with multiple IoU thresholds.
+The output of the previous detector is fed to the next as a **resampling** mechanism.
+![](/deep-learning/images/cascade-rcnn/fig3.png)
```

Comment:
add a space before and after this figure.  This will all the text to better render